src/NuGet.Clients/NuGet.CommandLine/Commands/InitCommand.cs

```diff
@@ -30,7 +31,7 @@ public class InitCommand : Command
             // If the Destination Feed Folder does not exist, it will be created.
             OfflineFeedUtility.ThrowIfInvalid(destination);
 
-            var packagePaths = GetPackageFilePaths(source, "*" + ProjectManagement.Constants.PackageExtension);
+            var packagePaths = GetPackageFilePaths(source, "*" + PackagingCoreConstants.NupkgExtension);
```

Comment:
But why can't it just be one file? Is it the library structure? The problem I have with the naming is that the word CORE is always meaningless